internal/cmd/egctl/translate.go

```diff
@@ -357,8 +357,8 @@ func printOutput(w io.Writer, result TranslationResult, output string) error {
 	if err != nil {
 		return err
 	}
-	fmt.Fprintln(w, string(out))
-	return nil
+	_, ret := fmt.Fprintln(w, string(out))
```

Comment:
why call it `ret` instead of reusing `err`?

Reply:
thought using `err` could cause a little confusing, but if not, we can still reuse `err`

Reply:
fixed